packages/atomic/src/components/common/tabs/tab-popover.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  createPopperLite as createPopper,
+  preventOverflow,
+  Instance as PopperInstance,
+} from '@popperjs/core';
+import {
+  Component,
+  h,
+  Listen,
+  State,
+  Element,
+  Host,
+  Prop,
+  Method,
+} from '@stencil/core';
+import ArrowBottomIcon from '../../../images/arrow-bottom-rounded.svg';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Bindings} from '../../search/atomic-search-interface/atomic-search-interface';
+import {Button} from '../button';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'tab-popover',
+  styleUrl: 'tab-popover.pcss',
+})
+export class TabPopover implements InitializableComponent {
+  @Element() private host!: HTMLElement;
+
+  @InitializeBindings() public bindings!: Bindings;
+
+  @Prop()
+  public hide = false;
+
+  @State()
+  public error!: Error;
+
+  @State()
+  private isOpen = false;
+
+  private buttonRef!: HTMLElement;
+  private popupRef!: HTMLElement;
+  private popperInstance?: PopperInstance;
+  public popoverId = 'tab-popover';
+
+  public initialize() {}
+
+  private initializePopover() {
+    this.popupRef.classList.add('popover-nested');
+  }
+
+  @Listen('keydown')
+  public handleKeyDown(e: KeyboardEvent) {
+    if (e.key === 'Escape' && this.isOpen) {
+      this.togglePopover();
+    }
+  }
+
+  get slotElements() {
+    return this.host.children;
+  }
+
+  get hasTabs() {
+    return !!this.popupRef.children.length;
+  }
+
+  @Method()
+  public togglePopover() {
+    this.isOpen = !this.isOpen;
+  }
+
+  private renderDropdownButton() {
+    const label = this.bindings?.i18n.t('more');
+    const ariaLabel = this.bindings?.i18n.t('popover', {label});
+    const buttonClasses = ['relative', 'pb-1', 'mt-1', 'mr-6', 'font-semibold'];
+
+    return (
+      <Button
```

Comment:
I would have been neat to be able to re-use the `IconButton` component, since it should properly handle accessibility concerns. 

However I guess it does not work exactly as you want here with what you are trying to do here with the label 🤔

Reply:
Yeah and it has a different look we'd need to override. I think this is simpler in this case.